### PR TITLE
update to v0.9.38

### PR DESCRIPTION
**Pull request by tgmichel**

### CODE REVIEW

File: pallets/aura-style-filter/src/lib.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@
 //!    Whereas sc-consensus-aura includes the entire consensus stack including block signing, digest
 //!    formats, and slot prediction. This is a lot of overhead for a sipmle round robin
 //!    consensus that basically boils down to this function
-//!    https://github.com/paritytech/substrate/blob/0f849efc/client/consensus/aura/src/lib.rs#L91-L106
+//!    https://github.com/purestake/substrate/blob/0f849efc/client/consensus/aura/src/lib.rs#L91-L106
```
Comment (JoshOrndorff) [inline pallets/aura-style-filter/src/lib.rs]: The dependencies on `main` still point to paritytech, maybe this link should too?

File: pallets/author-inherent/src/exec.rs [hunk 2]
```diff
@@ -35,7 +35,7 @@ use sp_runtime::{generic::DigestItem, RuntimeAppPublic};
 /// TODO Degisn improvement:
 /// Can we share code with the verifier?
 /// Can this struct take a verifier as an associated type?
-/// Or maybe this will just get simpler in general when https://github.com/paritytech/polkadot/issues/2888 lands
+/// Or maybe this will just get simpler in general when https://github.com/purestake/polkadot/issues/2888 lands
```
Comment (JoshOrndorff) [inline pallets/author-inherent/src/exec.rs]: This link is invalid

File: pallets/author-inherent/src/lib.rs [hunk 3]
```diff
@@ -116,7 +116,7 @@ pub mod pallet {
 	impl<T: Config> Pallet<T> {
 		/// This inherent is a workaround to run code after the "real" inherents have executed,
 		/// but before transactions are executed.
-		// This should go into on_post_inherents when it is ready https://github.com/paritytech/substrate/pull/10128
+		// This should go into on_post_inherents when it is ready https://github.com/purestake/substrate/pull/10128
```
Comment (JoshOrndorff) [inline pallets/author-inherent/src/lib.rs]: :eyes:

File: parachain-template/node/Cargo.toml [hunk 4]
```diff
@@ -6,7 +6,7 @@ description = "A Substrate node that demonstrates using the Nimbus consensus fra
 edition = "2021"
 homepage = "https://substrate.dev"
 license = "Unlicense"
-repository = "https://github.com/paritytech/cumulus/"
+repository = "https://github.com/purestake/cumulus/"
```
Comment (JoshOrndorff) [inline parachain-template/node/Cargo.toml]: :eyes:

File: parachain-template/node/src/command.rs [hunk 5]
```diff
@@ -55,7 +55,7 @@ impl SubstrateCli for Cli {
 	}
 
 	fn support_url() -> String {
-		"https://github.com/paritytech/cumulus/issues/new".into()
+		"https://github.com/purestake/cumulus/issues/new".into()
```
Comment (JoshOrndorff) [inline parachain-template/node/src/command.rs]: I'm starting to think these were from a search and replace

File: parachain-template/node/src/service.rs [hunk 6]
```diff
@@ -39,6 +40,7 @@ use sp_api::ConstructRuntimeApi;
 use sp_keystore::SyncCryptoStorePtr;
 use sp_runtime::traits::BlakeTwo256;
 use substrate_prometheus_endpoint::Registry;
+// use sp_client_db::Backend;
```
Comment (JoshOrndorff) [inline parachain-template/node/src/service.rs]: commented code

**Comment by tgmichel** [comment]
@librelois PTAL whenever you have a moment

**Comment by tgmichel** [comment]
@librelois @JoshOrndorff can I get a review on this so we can move one please?

**Review by librelois — APPROVED** [review]

**Comment by librelois** [comment]
@tgmichel LGTM

**Pull request merged**